src/pages/components/checkbox/usage.mdx

```diff
@@ -37,35 +52,191 @@ The default view of a set of checkboxes is having no option selected.
+
+<Row>
+<Column colLg={8}>
+
+![The default state for checkboxes is unchecked.](images/checkbox-usage-4.png)
+
+</Column>
+</Row>
+
+<Caption>Vertically stacked versus horizontal alignment.</Caption>
+
+### Placement
+
+The checkbox component is often used in forms. Forms can be placed on a full page, in a modal or in a side panel. A checkbox can also be used for agreeing to terms and conditions or to filter information.
+
+<Row>
+<Column colLg={8}>
+
+![The default state for checkboxes is unchecked.](images/checkbox-usage-5.png)
+
+</Column>
+</Row>
+
+Checkboxes in a form should be placed at least 32px (`layout-03`) below or before the next component. Having 24px or 16px can also be used when space is more restricted or if the form is more complex.
 
-## Related components
+For more information on spacing in forms, see our [form style guidance](/components/form/style).
 
-#### Checkbox vs. radio button
+<Row>
+<Column colLg={8}>
+
+![The default state for checkboxes is unchecked.](images/checkbox-usage-6.png)
+
+</Column>
+</Row>
 
-Whereas radio buttons represent a group of _mutually exclusive_ choices, users can select one or more checkboxes from a group. In use cases where only one selection of a group is allowed, use the radio button component instead of the checkbox.
+<Caption>Spacing between a checkbox and other components in a form.</Caption>
 
-#### Checkbox vs. toggle switch
+For more information about spacing for the checkbox component itself, see the [style tab](/components/checkbox/style).
 
-Generally, toggle switches are preferred when the resulting action will be instantaneously applied, without the need for further confirmation. Checkboxes generally represent one input in a larger flow which requires a final confirmation step.
+## Behavior
 
-#### References
+### States
 
-- [Toggle](/components/toggle/code)
-- [Radio button](/components/radio-button/code)
-- [Form](/components/form/code)
-- [Data table](/components/data-table/code)
+The checkbox control allows for a series of states: **unselected**, **selected**, and **indeterminate**. The default view of a set of checkboxes is having no option selected.
+
+<Row>
+<Column colLg={8}>
+
+![The default state for checkboxes is unchecked.](images/checkbox-usage-7.png)
+
+</Column>
+</Row>
+
+Use the indeterminate state when the checkbox contains a sublist of selections, some of which are selected, and some unselected. In addition to unselected, selected, and indeterminate states, checkboxes also have states for focus and disabled.
+
+<Row>
+<Column colLg={8}>
+
+![The default state for checkboxes is unchecked.](images/checkbox-usage-8.png)
+
+</Column>
+</Row>
+
+### Interactions
+
+#### Click target
+
+Users can select an item by clicking on the checkbox control directly or by clicking on the checkbox label. Having both regions interactive creates a more accessible click target.
```

Comment:
I think the click target area for this might be off by a bit, unfortunately I think the space around the checkbox is margin currently:

![image](https://user-images.githubusercontent.com/3901764/79907568-685b9700-83df-11ea-8c95-c8ce522fa16c.png)

And so wouldn't be interactive 😞 

 I think the click target for the text would be the line-height (24px?). The image could 100% have that exactly, just couldn't tell 😅 

Similarly, if we wanted to do guidance for how to interact with keyboard/SR we could use the same copy from accordion since I believe it's space/enter along with ctrl+option+space!

Reply:
OH okay, so the click target for the control is just the icon bounds only (not including the margin). And the click target for the label would be the line height.

If you think it would be helpful to add the keyboard guidance we can! Is it the exact same copy from Accordion or does it need to be tweaked?

Reply:
I think it's the same, yeah! The only change would be the focus outline placement.